src/plccng/scanner/matcher_test.py

```diff
@@ -0,0 +1,130 @@
+from plccng.lineparse import Line
+from .Skip import Skip
+from .Token import Token
+from .LexError import LexError
+
+from . import matcher
+
+
+def test_empty_line():
+    matcher = make_matcher(spec=[
+        {
```

Comment:
Using the structures in `specparse` will also allow us to use `specparse.lexparse.fromstring` in tests.